src/cascade/executor/no_covariate_main.py

```diff
@@ -259,10 +257,28 @@ def construct_database():
 
     # Get the bundle and process it.
     raw_inputs = data_from_csv(Path("measure.csv"))
-    model = internal_model(model_context, raw_inputs)
-    write_to_file(model, "fit.db")
+    internal_model(model_context, raw_inputs)
+
+    output_file = "fit.db"
+    LOGGER.info("Creating file {output_file}")
+    dismod_file = model_to_dismod_file(model_context, output_file)
+    flush_begin = timer()
+    dismod_file.flush()
+    LOGGER.debug(f"Flush db {timer() - flush_begin}")
+
+
+def entry():
+    """This is the entry that setuptools turns into an installed program."""
+    parser = ArgumentParser("Reads csv for a run without covariates.")
+    parser.add_argument("-v", help="increase debugging verbosity", action="store_true")
```

Comment:
If we're going to do this as an entrypoint we must let them supply the names of the input and output files.